code/game/mecha/mecha.dm

```diff
@@ -4,7 +4,8 @@
 	icon = 'icons/mecha/mecha.dmi'
 	density = TRUE //Dense. To raise the heat.
 	opacity = 1 ///opaque. Menacing.
-	anchored = TRUE //no pulling around.
+	move_force = MOVE_FORCE_VERY_STRONG

```

Comment:
Isn't that door ripping level

Reply:
Yes, but it still checks if the object isn't anchored first.